README.md

```diff
@@ -505,6 +505,18 @@ fastify.post('/upload/files', {
 })
 ```
 
+## Zod Schema body validation
+
+To validate requests using [Zod](https://github.com/colinhacks/zod), you need to:
+
+1. Install and configure [`fastify-type-provider-zod`](https://github.com/turkerdev/fastify-type-provider-zod).
+2. Make sure the `attachFieldsToBody` option is set to `true` when registering the `@fastify/multipart` plugin.
+3. You can use `attachFieldsToBody: "keyValues"` to avoid another fields preprocessing, but in that case, you will receive a Buffer for files that are not text/plain.
```

Comment:
```suggestion
1. Install and configure [`fastify-type-provider-zod`](https://github.com/turkerdev/fastify-type-provider-zod).
1. Make sure the `attachFieldsToBody` option is set to `true` when registering the `@fastify/multipart` plugin.
1. You can use `attachFieldsToBody: "keyValues"` to avoid another fields preprocessing, but in that case, you will receive a Buffer for files that are not text/plain.
```